source/channels/join-leave-channels.rst

```diff
@@ -16,11 +16,16 @@ To join a private channel, you need to be added to the channel by a member of th
 
   .. tab:: Desktop
 
-    1. Select the **+** symbol at the top of the channel sidebar, then select **Browse Channels**. 
+    1. Select the **Add channels** button in the channel sidebar, then select **Browse Channels**.
+     
+      .. image:: ../images/add-channels-button.png
+        :alt: You can browse channels using the Add channels button
```

Comment:
Please remove the red outline from this screenshot (to align with our style guide).

Also, I've confirmed in a local build that this image appears quite large compared to the next image on the same page. 

![image](https://user-images.githubusercontent.com/74422101/236297450-6e177569-06c3-4dda-b1e8-b5a7b87dd056.png)

Please add the following line to resize the image directly following the ``alt`` line:

```
:width: 400
```

This line will resize the image to a max width of 400 pixels. Please apply these updates to both instances of the image (here and the join/leave page).

Reply:
Ok, @cwarnermm. I will work on this next week.